README.md

```diff
@@ -17,8 +17,13 @@ $ npm install eslint-visitor-keys
 
 ### Requirements
 
+Since eslint-visitor-keys v3:
+- [Node.js] 12.22.0 or 14.17.0 or later.
+
+Before:
 - [Node.js] 10.0.0 or later.
```

Comment:
```suggestion
- [Node.js] `^12.22.0`, `^14.17.0`, or `>=16.0.0`
```

I think it's better to use this format, because 13 and 15 are not supported.

Also, there's no need to document requirements for previous versions, it's available in the history.